src/models/model.h

```diff
@@ -46,6 +46,8 @@ struct State {
   std::vector<std::string> adapter_names_;
   std::vector<OrtValue*> inputs_, outputs_;
 
+  std::vector<std::pair<std::string, std::string>> _ep_dynamic_options_next_run;
```

Comment:
nit: consistent with other members.

```suggestion
  std::vector<std::pair<std::string, std::string>> ep_dynamic_options_next_run_;
```

Reply:
Done.